.github/workflows/test_gen.yaml

```diff
@@ -333,7 +333,7 @@ jobs:
     needs:
     - build-jar
     runs-on: ubuntu-latest
-    timeout-minutes: 45
+    timeout-minutes: 135
```

Comment:
The time required is about 50 to 60 minutes, for example: https://github.com/featurebyte/featurebyte/actions/runs/7322531386/job/19944719602?pr=2013